planner/core/physical_plan_test.go

```diff
@@ -1570,3 +1570,71 @@ func (s *testPlanSuite) TestIndexJoinHint(c *C) {
 		}
 	}
 }
+
+func (s *testPlanSuite) TestAggregationHints(c *C) {
```

Comment:
Add a test case which contains subquery?

Reply:
For the first one, because `possibleChildProperties` are only **possible**... We'd better not rely too much on it, I handle this in line 1272 - 1275, you can take a look.

For the test case, I will add them soon.

Reply:
When adding test case, find another bug. Fix it soon...